### DISCUSSION TITLE
What work is required to add new architecture support (Win arm64)

### DISCUSSION THREAD

**Discussion by kfechter** [mono/mono#21782] [Q&A]
I just got a surface pro with the X Elite processor, and I'd like to contribute to open source projects to bring more first-class support for Windows Arm64 machines. I'm just curious what would be required to add the new architecture (even if community supported), to mono.

**Comment by lextm** [comment]
Both .NET Framework 4.8.1 and .NET 8/9 work very well there, so you might find something else to make better use of your time.
